Client/Frontend/Settings/HomepageSettings/HomePageSettingViewController.swift

```diff
@@ -207,15 +195,15 @@ class HomePageSettingViewController: SettingsTableViewController, FeatureFlagsPr
 
 // MARK: - TopSitesSettings
 extension HomePageSettingViewController {
-    class TopSitesSettings: Setting, FeatureFlagsProtocol {
+    class TopSitesSettings: Setting, FeatureFlaggable {
         var profile: Profile
 
         override var accessoryType: UITableViewCell.AccessoryType { return .disclosureIndicator }
         override var accessibilityIdentifier: String? { return AccessibilityIdentifiers.Settings.Homepage.CustomizeFirefox.Shortcuts.settingsPage }
         override var style: UITableViewCell.CellStyle { return .value1 }
 
         override var status: NSAttributedString {
-            let areShortcutsOn = featureFlags.userPreferenceFor(.topSites) == UserFeaturePreference.enabled
+            let areShortcutsOn = featureFlags.isFeatureEnabled(.topSites, checking: .userOnly)
```

Comment:
Marking this as I might want to chat about it quickly tomorrow with Laurie and don't wanna forget.